### PR TITLE
add unsigned to the new migrations

### LINKED ISSUE #9443: Add support for unsigned integer to migration schema builder
Exception 'yii\base\UnknownMethodException' with message 'Calling unknown method: yii\db\ColumnSchemaBuilder::unsigned()'

in /var/www/ivm/vendor/yiisoft/yii2/base/Object.php:220

My code:

	public function up() {
		$this->createTable('auditLog', [
			'id' => $this->primaryKey(),
			'model' => $this->string(255)->notNull(),
			'action' => $this->string(255)->notNull(),
			'oldAttributes' => $this->text(),
			'newAttributes' => $this->text(),
			'IPAddress' => $this->string(255),
			'createdAt' => $this->datetime(),
			'updatedAt' => $this->datetime(),
			'createdBy' => $this->integer(10)->unsigned(),
			'updatedBy' => $this->integer(10)->unsigned(),
		]);
	}

### PR DESCRIPTION
**Pull request by berezuev**
resolves #9443

### CODE REVIEW

**Comment by vaseninm** [comment]
It is not support pgsql database.

**Comment by berezuev** [comment]
@vaseninm how can I get database type?
I can add expression for mysql

**Comment by vaseninm** [comment]
@berezuev namespace yii\db\mysql; For example: framework/db/oci/ColumnSchemaBuilder.php

File: framework/db/ColumnSchemaBuilder.php [hunk 1]
```diff
@@ -151,6 +167,18 @@ protected function buildUniqueString()
...
         return $this->isUnique ? ' UNIQUE' : '';
     }
+    
+    /**
+     * Builds the unsigned constraint for the column.
+     * @return string returns string 'UNSIGNED' if [[isUnsigned]] is true, otherwise it returns an empty string.
+     */
+    protected function buildUnsignedString()
+    {
+        if (gettype($this->default) == 'string') {
```
Comment (cebe) [inline framework/db/ColumnSchemaBuilder.php]: what is this line used for?

Comment (berezuev) [inline framework/db/ColumnSchemaBuilder.php]: string columns can't be unsigned.

**Comment by samdark** [comment]
Can't be applied because incompatible with at least PostgreSQL.

**Pull request closed without merging**